frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedPanel.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { getActiveNamespace } from '@console/internal/actions/ui';
+import { ResourceIcon, ResourceLink, SimpleTabNav } from '@console/internal/components/utils';
+import { referenceFor } from '@console/internal/module/k8s';
+import TopologyOperatorBackedResources from './TopologyOperatorBackedResources';
+import { TopologyDataObject } from '../topology-types';
+
+export type TopologyResourcePanelProps = {
+  item: TopologyDataObject<string>;
+};
+
+const TopologyOperatorBackedPanel: React.FC<TopologyResourcePanelProps> = ({ item }) => {
+  const {
+    name,
+    resources: { obj },
+  } = item;
+
+  const ResourcesSection = () => <TopologyOperatorBackedResources item={item} />;
+
+  return (
+    <div className="overview__sidebar-pane resource-overview">
+      <div className="overview__sidebar-pane-head resource-overview__heading">
+        <h1 className="co-m-pane__heading">
+          <div className="co-m-pane__name co-resource-item">
+            <ResourceIcon kind="Operator" />
```

Comment:
can use model to get kind or pick from obj itself, wdyt?

Reply:
I added a separate icon because `ResourceLink` was showing a `CSV` icon, but design docs had `O`, and `O` is also consistent with topology node. I can change it if it's not required.